src/Conticket/ApiBundle/Controller/EventController.php

```diff
@@ -62,49 +69,52 @@ public function cgetAction($limit, $offset)
      */
     public function getAction($id)
     {
-        return ['event' => $this->getOr404($id)];
```

Comment:
@abdala I will finish this until tomorrow. I need to allow behat tests on travis too